issue found on: temp/analyser.go:181:1: paramTypeCombine

an issue has been detected in source code: ./temp/analyser.go:181:1: paramTypeCombine: func(entries []AggregatorLogEntry, notRead []AggregatorLogEntry) could be replaced with func(entries, notRead []AggregatorLogEntry)

--- a/temp/analyser.go
+++ b/temp/analyser.go
@@ -177,7 +177,7 @@ func printErrorsForMessageWithOffset(entries []AggregatorLogEntry, offset int) {
 	}
 }
 
-func printConsumedEntries(entries []AggregatorLogEntry, notRead []AggregatorLogEntry) {
+func printConsumedEntries(entries, notRead []AggregatorLogEntry) {
 	for _, entry := range notRead {
 		printConsumedEntry(entry)
 		printErrorsForMessageWithOffset(entries, entry.Offset)

